hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -621,25 +621,6 @@ private void flushToDiskIfRequired(HoodieRecord record, boolean appendDeleteBloc
     }
   }
 
-  private HoodieLogBlock.HoodieLogBlockType pickLogDataBlockFormat() {
-    Option<HoodieLogBlock.HoodieLogBlockType> logBlockTypeOpt = config.getLogDataBlockFormat();
-    if (logBlockTypeOpt.isPresent()) {
-      return logBlockTypeOpt.get();
-    }
-
-    // Fallback to deduce data-block type based on the base file format
-    switch (hoodieTable.getBaseFileFormat()) {
-      case PARQUET:
-      case ORC:
-        return HoodieLogBlock.HoodieLogBlockType.AVRO_DATA_BLOCK;
-      case HFILE:
```

Comment:
Is this going to enable Avro log block for MDT?  MDT has to use HFile log blocks for efficient skipping.

Reply:
If not necessary, let's avoid the aggressive change and keep the existing behavior.

Reply:
This shouldn't have any behavior change. I just moved it so that the code is reusable

Reply:
Got it.